media-libs/libsoundtouch/libsoundtouch-2.1.0.recipe

```diff
@@ -10,30 +10,40 @@ than originally without affecting the sound pitch.
 tempo (speed).
 Playback Rate : Changes both tempo and pitch together as if a vinyl disc was \
 played at different RPM rate."
-HOMEPAGE="http://www.surina.net/soundtouch"
-COPYRIGHT="2001-2014 Olli Parviainen"
+HOMEPAGE="https://www.surina.net/soundtouch/"
+COPYRIGHT="2001-2018 Olli Parviainen"
 LICENSE="GNU LGPL v2.1"
 REVISION="1"
-SOURCE_URI="http://www.surina.net/soundtouch/soundtouch-$portVersion.tar.gz"
-CHECKSUM_SHA256="d224f7d1421b5f8e74a74c85741345bd9802618a40ae30ce5b427a5705c89d25"
-SOURCE_DIR="soundtouch"
-PATCHES="libsoundtouch-$portVersion.patchset"
+SOURCE_URI="https://gitlab.com/soundtouch/soundtouch/-/archive/$portVersion/soundtouch-$portVersion.tar.bz2"
+CHECKSUM_SHA256="3fb740c7ac4b304342fbf11a91d14fa3e296269b9fec6266609c4620477d0ad6"
+SOURCE_DIR="soundtouch-$portVersion"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="1.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVersionCompat="$portVersion compat >= 1"
+
 PROVIDES="
-	libsoundtouch$secondaryArchSuffix = $portVersion compat >= 1
-	cmd:soundstretch$secondaryArchSuffix
-	lib:libSoundTouch$secondaryArchSuffix = 1.0.0 compat >= 1
+	libsoundtouch$secondaryArchSuffix = $portVersionCompat
+	cmd:soundstretch$commandSuffix = $portVersion compat >= 2.1
```

Comment:
I just checked that there are no recipes depending on `cmd:soundstretch` or `libsoundstretch`, so this means that using the same compat string for both the package and the `cmd:soundstretch` would have no impact:
```diff
-portVersionCompat="$portVersion compat >= 1"
+portVersionCompat="$portVersion compat >= 2.1"
 
 PROVIDES="
 	libsoundtouch$secondaryArchSuffix = $portVersionCompat
-	cmd:soundstretch$commandSuffix = $portVersion compat >= 2.1
+	cmd:soundstretch$commandSuffix = $portVersionCompat
```